src/Lang/Cpp/CppParser.ts

```diff
@@ -428,6 +428,13 @@ export default class CppParser implements ICodeParser {
                 return "";
             } else if (nextLineTxt.startsWith("#") && ! nextLineTxt.startsWith("#define")) {
                 return "";
+            } else if (nextLine.line === 2) { // Check if there where two empty lines trailing the file
+                if (this.activeEditor.document.lineAt(0).text === this.cfg.C.firstLine
+                    && this.activeEditor.document.lineAt(1).text === this.cfg.C.commentPrefix
+                    && this.activeEditor.document.lineAt(2).text === "") {
```

Comment:
```suggestion
                    && (
                        this.activeEditor.document.lineAt(1).text === this.cfg.C.commentPrefix
                        || this.activeEditor.document.lineAt(1).text.trim() === ""
                     ) && this.activeEditor.document.lineAt(2).text.trim() === "") {
```
`this.activeEditor.document.lineAt(1).text` is `""` for me and not the content of the `commentPrefix` setting.
Do you have an idea why the two installations are different?
We can work around it with the suggestion.

Reply:
That's caused by the very common extension https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools.

Reply:
Right that makes sense.
My test build has no extensions installed but this one.